pkg/ottl/parser_test.go

```diff
@@ -2542,6 +2546,7 @@ func Test_Statement_Execute(t *testing.T) {
 		function          ExprFunc[any]
 		expectedCondition bool
 		expectedResult    any
+		expectedSpan      expectedSpan
```

Comment:
you define that field, and set it. Where do you compare it to the span of the run? same for next tests.